docs/evaluation.md

```diff
@@ -33,21 +33,52 @@ agent = AnyAgent.create(
     tracing=TracingConfig(output_dir="traces")
 )
 
-agent.run("How many seconds would it take for a leopard at full speed to run through Pont des Arts?")
+agent_trace = agent.run("How many seconds would it take for a leopard at full speed to run through Pont des Arts?")
+# If doing an evaluation via a separate python process, the trace can also be dumped to a json file:
+# with open("agent_trace.json", "w", encoding="utf-8") as f:
+#     f.write(agent_trace.model_dump_json(indent=2))
 ```
-1. Define a test case in a yaml file, e.g.
 
 
-~~~yaml
-{% include "./examples/test_case.yaml" %}
-~~~
+### Define an evaluation case either in a yaml file or in python:
+
+=== "YAML"
+    ~~~yaml
+    {% include "./examples/evaluation_case.yaml" %}
+    ~~~
+    Then in python
+    ```python
+    from any_agent.evaluation.evaluation_case import EvaluationCase
+    evaluation_case = EvaluationCase.from_yaml(evaluation_case_path)
+    ```
+
+=== "Python"
+    ```python
+    from any_agent.evaluation.evaluation_case import EvaluationCase
+    evaluation_case = EvaluationCase(
+            ground_truth=[{"name": "Test Case 1", "value": 1.0, "points": 1.0}],
+            checkpoints=[{"criteria": "Check if value is 1.0", "points": 1}],
+            llm_judge="gpt-4o-mini",
+            final_answer_criteria=[]
+    )
+    ```
+
+### Run the evaluation using the test case and trace.
 
-1. Run the evaluation using the test case and trace.
 ```python
-from any_agent.evaluation.test_case import TestCase
-from any_agent.evaluation.evaluate import evaluate_telemetry
-test_case = TestCase.from_yaml("./docs/examples/test_case.yaml")
-evaluate_telemetry(test_case, '/path/to/telemetry/output')
+from any_agent.evaluation import EvaluationRunner
+from any_agent.evaluation.evaluation_case import EvaluationCase
+output_path="tmp/path/result.json"
+evaluation_case = EvaluationCase(
+    ground_truth=[{"name": "Test Case 1", "value": 1.0, "points": 1.0}],
+    checkpoints=[{"criteria": "Check if value is 1.0", "points": 1}],
+    llm_judge="gpt-4o-mini",
+    final_answer_criteria=[]
```

Comment:
💅 Maybe use `final_output` instead of `final_answer` for consistency with the name of the AgentTrace property